doc/reference/configuration/configuration_reference.rst

```diff
@@ -3133,6 +3133,35 @@ log.syslog.*
     | Default: box.NULL
     | Environment variable: TT_LOG_SYSLOG_SERVER
 
+.. _configuration_reference_lua:
+
+lua
+---
+
+The ``lua`` section outlines the configuration parameters related to the ``lua`` environment within Tarantool.
+
+Below is an overview of the available configuration options for ``lua``:
+
+- :ref:`lua.memory <configuration_reference_lua>`
+
+
+.. _configuration_reference_lua_memory:
+
+.. confval:: lua.memory
+
+Specifies the amount of memory allocated to ``lua``, measured in bytes.
+
+- **Type**: integer
+- **Default Value**: 2GB
```

Comment:
Since it's a number, lets also show the exact value in bytes. Find similar examples in other config options.